### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by andylupu** [Leo-Corporation/Gavilya#251] [Q&A]
How are the games stored , after the user adds it on the app ?

**Comment by lpeyr** [comment]
Gavilya stores a game's information in a `GameInfo` class. This `GameInfo` class is then stores in a `List` (in the `Definitions` class) which will be saved in a `.gav` file using XML serialization. This file is available in `%APPDATA%\Gavilya`. 
This process is handled by the `GameSaver` class:
https://github.com/Leo-Corporation/Gavilya/blob/ec7d8a6b7477c23724c3e45fb286b724f7575c5f/Gavilya/Classes/GameSaver.cs#L38-L53

The `List` where the games are stored:
https://github.com/Leo-Corporation/Gavilya/blob/ec7d8a6b7477c23724c3e45fb286b724f7575c5f/Gavilya/Classes/Definitions.cs#L158-L161

The `GameInfo` class:
https://github.com/Leo-Corporation/Gavilya/blob/ec7d8a6b7477c23724c3e45fb286b724f7575c5f/Gavilya/Classes/GameInfo.cs#L31-L108
